src/rendition-mixin.js

```diff
@@ -54,21 +54,16 @@ class Representation {
                               .fastQualityChange_
                               .bind(hlsHandler.masterPlaylistController_);
 
-    // Carefully descend into the playlist's attributes since most
-    // properties are optional
-    if (playlist.attributes) {
-      let attributes = playlist.attributes;
+    // some playlist attributes are optional
+    if (playlist.attributes.RESOLUTION) {
+      let resolution = playlist.attributes.RESOLUTION;
```

Comment:
Kind of a nitpick, by why let and not const?  If statement is the scope and the value doesn't seem to change.